s3/Acornfile

```diff
@@ -7,9 +7,34 @@ args: {
 	tags: {}
 }
 
-services: {
-	"s3-bucket": generated: job: "apply"
-	"cdk-context": build: "../utils/cdk"
+services: "s3-bucket-read": {
```

Comment:
Based on the SQS discussions, we should shorten the names to just the function and add one more:
`readonly`
lets add this one and make it the default.
`readwrite` -> Make the default
`writeonly`
`admin`

Reply:
Should deletion be included along with the write permissions or is that reserved for admin usage?